src/utils/src/deviceinfo.cpp

```diff
@@ -16,49 +16,49 @@
  */
 
 #include "deviceinfo.h"
+#include <QFile>
+#include <QCoreApplication>
+
+const char* CONFIG_FILE = "/etc/machine.conf";
+
+DeviceInfo::DeviceInfo()
+{
+    QCoreApplication::setOrganizationName("AsteroidOS");
+    QCoreApplication::setOrganizationDomain("asteroidos.org");
+    QCoreApplication::setApplicationName("Device Info");
```

Comment:
Mhhh why is this necessary ? Aren't those set in here already https://github.com/AsteroidOS/qml-asteroid/blob/master/src/app/asteroidapp.cpp#L65 ?
DeviceInfo should not set the application name of a global singleton to "Device Info".

Reply:
If lipstick doesn't set those, then those should be set in Lipstick, not in a QML Item

Reply:
You are correct, this wasn't necessary.